<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CmdWidget</class>
 <widget class="QWidget" name="CmdWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>450</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QComboBox" name="comboBox">
              <property name="minimumSize">
               <size>
                <width>400</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <item>
               <property name="text">
                <string>dir</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>adb shell getprop</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>adb shell pm list packages</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>adb</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButton_Execute">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Execute</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_Stop">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Stop</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder></zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
